example-build/tests/wasm.rs

```diff
@@ -0,0 +1,11 @@
+use wasm_bindgen_test::*;
+
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
+
+include!(concat!(env!("OUT_DIR"), "/codegen.rs"));
+
+#[wasm_bindgen_test]
+fn test_client_new() {
+    let client = Client::new("http://foo/bar");
+    assert!(client.baseurl == "http://foo/bar");
+}
```

Comment:
the purpose of the examples is for people to copy to them; let's keep the tests out of here.

Reply:
I agree, moved to `example-wasm`!